README.md

```diff
@@ -13,9 +13,9 @@
         git clone https://github.com/gazebo-web/gazebosim-web-frontend
         cd web
 
-1. This repository comes with an example `setup.bash` file which works with the
+1. Add a `setup.bash` file which works with the
    API at https://staging-api.gazebosim.org. Edit that file with
-   your own Auth0 and backend information. Always remember to source this file
+   your own Auth0 and backend information i.e. the environment variables set while [installing the backend](https://github.com/gazebo-web/gazebosim-web-backend#install). Always remember to source this file
```

Comment:
In my testing, I didn't have to set any environment variables in the frontend. Do you remember which variables you had to set?

Reply:
Is this the frontend.bash? I wasn't able to load the Docs page even after using that file :( Before I used it though, sometimes it worked. It's always been flakey for me...

Reply:
@mabelzhang I encountered the issue you ran into with the page being empty. Rebuilding the docker image fixed it for me.

Reply:
The variables set in point 6 in https://github.com/gazebo-web/gazebosim-web-backend#install.

This with following the install intructions is something that worked for me, though I didn't go back and forth to test dependencies, will update once I try again.

Reply:
Oh damn, I just realized while reading this diff that my local path in docker-compose.yml has been overwritten probably from a pull :sweat_smile: That's probably why it hasn't worked for me lately -_-

Reply:
Ahh okay, I still need to try again. Meanwhile, these instructions definitely help when developing, so might be worth documenting.